Program.cs

```diff
@@ -89,8 +89,17 @@
             );
     }
 );
-
-builder.Services.AddHangfireServer();
+var isProd = Environment.GetEnvironmentVariable("IS_PROD");
+var queueName = Environment.GetEnvironmentVariable("QUEUE_NAME");
+if (isProd == "false")
+{
+    // create a queue for testing purposes
+    builder.Services.AddHangfireServer(op => op.Queues = [queueName]);
+}
```

Comment:
this is causing "obsolete" errors elsewhere in the code. could you give me the run down on why we need  to specify the queue name in the first place?

Reply:
I specify the queue name here so that a queue of whatever name we give it, `parker` for example, will be created. Then in the job service tell it what queue to use. If we are in prod it uses the standard `default` queue and when we are testing we have it create a custom queue so that we don't utilize the production one which could be running different code. Does that make sense?

Reply:
yep I gotcha, thanks